test/unit/test_describe.py

```diff
@@ -84,4 +91,4 @@ def test_describe(
         ]
         root_dir = exp.exp_path.parent
         for expid in expids:
-            assert f'Location: {str(root_dir / expid)}' in log_result_output
+            assert f'Location: {exp.exp_path}' in log_result_output
```

Comment:
With this change, `root_dir` is not used, nor `expid`. So you could simplify it with something like,

```py
    if exp and not not_described:
        log_result_output = [
            line_tuple.args[0].format(line_tuple.args[1]) for line_tuple in Log.result.mock_calls
        ]
        for _ in expids:
            assert f'Location: {exp.exp_path}' in log_result_output
```

Reply:
Note that in the future we will configure a linter to fail the build with changes like this, #2040 , but you can also configure IDEs to display this (as well as coverage) locally, e.g.

![image](https://github.com/user-attachments/assets/bd87d22b-1a5c-4546-af66-f3a47e5641ab)

This is displayed by default in PyCharm without any configuration. vim/emacs/etc. should have a plugin/macro that does something like this too, or you can configure linters locally.